src/libmscore/draw/drawbufferjson.cpp

```diff
@@ -39,31 +40,57 @@ static int rtoi(qreal v)
     return v * 1000;
 }
 
+static qreal itor(int v)
+{
+    return static_cast<qreal>(v) / 1000.0;
+}
+
 static QJsonObject toObj(const QPen& pen)
 {
-    QJsonObject o;
-    o["style"] = static_cast<int>(pen.style());
-    o["color"] = pen.color().name();
-    o["width"] = pen.width();
-    return o;
+    QJsonObject obj;
+    obj["style"] = static_cast<int>(pen.style());
```

Comment:
I think it is better to add static strings at the top of this file and use them as keys for serialization/deserialization

Reply:
Not a bad idea, but I think this is superfluously in this case.